x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/inference/assignment/TrainedModelAssignmentNodeService.java

```diff
@@ -146,24 +151,6 @@ public void beforeStop() {
         this.expressionResolver = expressionResolver;
     }
 
-    void stopDeploymentAsync(TrainedModelDeploymentTask task, String reason, ActionListener<Void> listener) {
```

Comment:
I moved this below and refactored it into a helper that takes a consumer function since the graceful and forceful stop deployments essentially the same thing